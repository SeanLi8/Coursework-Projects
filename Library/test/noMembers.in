member 100000
member books 100000
member history 100000
rent 100000 111111
relinquish 100000 111111
relinquish all 111111
common
exit