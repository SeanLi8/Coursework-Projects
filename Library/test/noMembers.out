user: No members in system.

user: No members in system.

user: No members in system.

user: No members in system.

user: No members in system.

user: No members in system.

user: No members in system.

user: Ending Library process.
